modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotFutureTask.java

```diff
@@ -512,9 +523,13 @@ else if (affNode && missed.isEmpty() && cctx.kernalContext().query().moduleEnabl
 
                                 snpSndr.sendDelta(delta, cacheDirName, pair);
 
+                                long deltaLen = delta.length();
```

Comment:
processedSize.addAndGet(delta.length()) ?

I think assertion can only be thrown during development?)